src/spikeinterface/core/binaryrecordingextractor.py

```diff
@@ -65,13 +65,9 @@ def __init__(
         gain_to_uV=None,
         offset_to_uV=None,
         is_filtered=None,
-        num_chan=None,
     ):
-        # This assigns num_channels if num_channels is not None, otherwise num_chan is assigned
-        num_channels = num_channels or num_chan
```

Comment:
@zm711 unfortunately, I think we need to keep this. This is not only a deprecation for a current version, but it allows to load recordings that were saved with old versions.

Reply:
Sounds good!